### ISSUE TITLE
Listen to request outside of localhost

### ISSUE THREAD

**Issue by alallier** [dularion/streama#69]
I currently have a port forward to streama on port 8080, so that I can access this outside my local network, how do I configure streama to allow this connection?

**Comment by dularion** [comment]
Well, let's assume your routers IP address is something like 93.126.189.103, in that case you would reach streama via http://93.126.189.103:8080/streama. Now all you need to do, is set that url as your base url in the application. If you leave the base url at localhost, the application won't be able to serve the videos correctly. I think that should work. Let me know if it doesn't

**Comment by alallier** [comment]
I tried that and couldn't get it to work. To test to make sure I could reach the vm outside of the network and isolate that it was streama I ran a python quick server `$ python -m SimpleHTTPServer 8080` and I could reach the server on localhost from outside the network using my public IP. So it is like streama won't listen to the connection if it's not localhost..? I did change the base url as you said too.
